eeg_ds000117/dataset_description.json

```
@@ -1,6 +1,7 @@
 {
 	"Name": "EEG derived data from 'A multi-subject, multi-modal human neuroimaging dataset: fMRI, EEG and MEG data on face processing'",
 	"BIDSVersion": "v1.X, BEP006",
+	"HEDVersion": "v8.0.0-alpha.1"
```

you are missing a comma here
```suggestion
	"HEDVersion": "v8.0.0-alpha.1",
```